### ISSUE TITLE
Make the library usable from Swift

### ISSUE THREAD

**Issue by mike-lischke** [gianlucabertani/Objective-Zip#23]
This would probably be automatically working if there were frameworks.

**Comment by olejnjak** [comment]
Making frameworks won't help as long as enums are just C enums (e.g. OZZipFileMode). Converting them to NS_ENUM would help.

**Comment by gianlucabertani** [comment]
You are right, it's an oversight. I'll fix it in 1.0.1, just waiting for a few more feedbacks updating.
